app/Http/Controllers/StatementController.php

```diff
@@ -176,5 +178,120 @@ public function getStatementHistory(Request $request, Validate $validate)
         }
     }
 
- 
+    public function editStatement($id)
+    {
+        $statement = [];
+        try {
+            $statement = Statement::where('id', $id)->first();
+            if ($statement) {
+                $filter['topicNum'] = $statement->topic_num;
+                $filter['campNum'] = $statement->camp_num;
+                $filter['asOf'] = 'default';
+                $topic = Camp::getAgreementTopic($filter);
+                $camp = Camp::getLiveCamp($filter);
+                $parentcampnum = isset($camp->parent_camp_num) ? $camp->parent_camp_num : 0;
+                $parentcamp = Camp::campNameWithAncestors($camp, $filter);
+                $nickName = Nickname::topicNicknameUsed($statement->topic_num);
+                $statement->go_live_time = date('m/d/Y, h:i:s A', $statement->go_live_time);
+                $WikiParser = new wikiParser;
+                $statement->parsed_value = $WikiParser->parse($statement->value);
+                $data = new stdClass();
+                $data->statement = $statement;
+                $data->topic = $topic;
+                $data->parent_camp = $parentcamp;
+                $data->nick_name = $nickName;
+                $data->parentcampnum = $parentcampnum;
+                $response[0]=$data;
+                $indexs = ['statement', 'topic', 'parent_camp', 'nick_name', 'parentcampnum'];
```

Comment:
By looking at the other elements of array `parentcampnum` should be `parent_camp_num`